tests/unit/setupUnitTest.ts

```diff
@@ -110,7 +110,8 @@ module.exports = async () => {
   const tstFilePath = path.join(vcastEnvPath, tstFilename);
   const createTstFile = `echo -- Environment: TEST > ${tstFilePath}`;
   {
-    const stderr = await promisifiedExec(createTstFile)[1];
+    const result = await promisifiedExec(createTstFile);
```

Comment:
Why not:

```
const { stdout, stderr } = await promisifiedExec(checkClicast);
```

?

Reply:
Refactored code in e692cf4675eb7ccf870c3d73c1df16f09988653c. Added `runCommand` function in `utils` to reduce code duplication.